packages/salesforcedx-vscode-core/src/statuses/taskView.ts

```diff
@@ -88,9 +88,6 @@ export class TaskViewService implements TreeDataProvider<Task> {
 }
 
 export class Task extends TreeItem {
-  public readonly label: string;
-  public readonly collapsibleState: TreeItemCollapsibleState;
```

Comment:
Why did we remove this? Was it unused?

Reply:
Yeah, these are completely unused.  The TreeItem takes them in the super() constructor.  So I got nervous about instantiating them here, since that might override TreeItem.label and TreeItem.collapsibleState in some way.